dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/features/AccountApiLiveTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.dimensiondata.cloudcontrol.features;
+
+import org.jclouds.dimensiondata.cloudcontrol.domain.Account;
+import org.jclouds.dimensiondata.cloudcontrol.internal.BaseDimensionDataCloudControlApiLiveTest;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.AssertJUnit.assertEquals;
+
+@Test(groups = "live", testName = "AccountApiLiveTest", singleThreaded = true)
+public class AccountApiLiveTest extends BaseDimensionDataCloudControlApiLiveTest {
+   @Test
+   public void testGetAccount() {
+      Account account = api().getMyAccount();
+      assertNotNull(account);
+      assertEquals("devlab1-online-vendor-devuser1", account.userName());
+      assertEquals("Donal Lunny", account.fullName());
+      assertEquals("Donal", account.firstName());
+      assertEquals("Lunny", account.lastName());
+      assertEquals("devlab1-online-vendor-devuser1@opsource.net", account.emailAddress());
+      assertThat(account.roles().contains(Account.RoleType.create("storage")));
+      assertThat(account.roles().contains(Account.RoleType.create("primary administrator")));
+      assertEquals("1", account.phone().countryCode());
+      assertEquals("1234567", account.phone().number());
+      assertEquals("6ac1e746-b1ea-4da5-a24e-caf1a978789d", account.organization().id());
+      assertEquals("Eng Customer of DevLab1-homed online sign-up Vendor", account.organization().name());
+      assertEquals("DevLab1Geo1", account.organization().homeGeoName());
+      assertEquals("apidevlab1.opsourcecloud.net", account.organization().homeGeoApiHost());
+      assertEquals("DEVLAB1GEO1", account.organization().homeGeoId());
+      assertEquals("NORMAL", account.state());
```

Comment:
We have 2 kinds ot tests in jclouds: mock and live tests.

Mock tests verify that we generate the requests we are expected to generate. One could think about them as "jclouds generates exactly what the API docs say".
Live tests are integration tests that perform real requests to the target cloud (and may cost money!). The idea is that anyone with an account should be able to run them to have the real status of the APIs, configure them in the CI to keep track of the changes on the provider API, etc.